dali/dfu/dfurun.cpp

```diff
@@ -1276,16 +1276,22 @@ class CDFUengine: public CInterface, implements IDFUengine
 
                     // keys default wrap for copy
                     if (destination->getWrap()||(iskey&&(cmd==DFUcmd_copy)))
-                        destination->setNumPartsOverride(srcFile->numParts());
-                    else if (isContainerized())
+                    {
+                        IDFUfileSpec *dst = wu->queryUpdateDestination();
```

Comment:
perhaps we should throw an error here if destination->getNumPartsOverride() has been set.

Reply:
I'm not sure that we should.  The destination->getNumPartsOverride() may be set by the FileSpray/SprayFixed or FileSpray/SpraryVariable to "override" the defaultSprayParts provide under the storage plane configuration.